.gitignore

```diff
@@ -73,3 +73,6 @@ tags
 
 # vim patches
 /vim-*.patch
+
+# ctest artifcats
+Testing
```

Comment:
can it go in `build/testing/` or something like that?

Reply:
I've still haven't had a chance to look into this, see https://github.com/neovim/neovim/pull/18674#discussion_r906810209

Reply:
I don't really understand what you mean? If this isn't needed for this PR then it shouldn't really be here.